http-server/src/main/java/org/triplea/server/user/account/login/authorizer/BCryptHashVerifier.java

```diff
@@ -1,14 +1,17 @@
 package org.triplea.server.user.account.login.authorizer;
 
+import at.favre.lib.crypto.bcrypt.BCrypt;
+import at.favre.lib.crypto.bcrypt.LongPasswordStrategies;
 import java.util.function.BiPredicate;
-import org.mindrot.jbcrypt.BCrypt;
 import org.triplea.server.user.account.PasswordBCrypter;
 
 public class BCryptHashVerifier implements BiPredicate<String, String> {
   @Override
   public boolean test(final String password, final String databaseBcrypted) {
     // TODO: Md5-Deprecation Move SHA512 hashing to client side
     final String hashedPassword = PasswordBCrypter.hashPasswordWithSalt(password);
-    return BCrypt.checkpw(hashedPassword, databaseBcrypted);
+    return BCrypt.verifyer(null, LongPasswordStrategies.none())
```

Comment:
The level of abstraction is perhaps inconsistent here, would you agree @RoiEXLab ?

Notably, we have a high level call `PasswordBCrypter.hashPasswordWithSalt`, which abstracts away parameters and configuration, then we have a more low level call to verify.

Second, the params to `verifyer` seem to be chosen for specific reasons, but it's not immediately clear (missing context) what those reasons were.

I think you could resolve both concerns by moving the `verifyer` call to a similar static method. That method could then be documented to note why the parameters are chosen as they are, and we'd have a consistent abstraction in this method. For example:
```
final String hashedPassword = PasswordBCrypter.hashPasswordWithSalt(password);
return BcryptedPasswordVerifyer.verify(hashedPassword);
```

WDYT @RoiEXLab ?

(A third benefit, we would not need to repeat the verifyer parameters in [test](https://github.com/triplea-game/triplea/pull/5822/files#diff-d9b0f1911d3493fb0dffedbe799f1d81R31), DRY)